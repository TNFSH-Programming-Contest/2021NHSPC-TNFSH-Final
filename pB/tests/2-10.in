8 16 8
1 5 10 15 13 4
10 15 13 5 12 3
11 5 15 12 16 4
12 14 15 16 13 4
10 9 4 8 1 14
3 14 9 15 2 7
13 3 7 5 6 4
7 3 11 8 16 6
2 1 5
1 6
4 6 7 8 2
5 4 7 5 1 3
4 3 2 1 7
2 7 8
3 7 8 6
2 5 8
2 6 5
3 2 5 1
2 3 8
3 3 2 4
4 4 2 1 7
3 6 5 4
5 3 6 1 4 2
3 3 4 8
