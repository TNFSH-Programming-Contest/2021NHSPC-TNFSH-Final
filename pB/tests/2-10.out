1 1
0
1 6
0
0
0
1 8
0
0
2 2 5
1 3
1 4
1 7
0
0
0
